js/ez-field-edit-author.js

```diff
@@ -0,0 +1,194 @@
+/* jshint esversion:6 */
+(function () {
+    const ROW_TEMPLATE = `
+        <div class="field__row">
```

Comment:
this does not follow the markup defined in https://github.com/ezsystems/hybrid-platform-ui/blob/master/docs/specs/field_edit_markup.md#authors-ezauthor

But in any case, this template can be not be defined inside the component because it will also be used by the server to generate the initial field edit markup when some authors are already in the field value. That also means (as in most components), we can not use the shadow DOM and we have to stick to the light DOM.